### PR TITLE
Add client-app-proxy command (a.k.a. cozysdk-shell / cozysdk-bench)

### PR DESCRIPTION
**Pull request by aenario**

### CODE REVIEW

File: src/main.coffee [hunk 1]
```diff
@@ -140,6 +140,39 @@ program
                 appUrl = "http://localhost:9104/#apps/#{app.slug}"
                 log.info "You can see your app on #{appUrl}"
 
+program
+.command "bench <path>"
```
Comment (nono) [inline src/main.coffee]: Why did you called it `bench`?

File: src/main.coffee [hunk 2]
```diff
@@ -140,6 +140,39 @@ program
...
+            exitHandler = (err) ->
+                console.log(err.stack) if err
+                return process.exit() if (secondExit)
+                console.log('cleaning up')
+                secondExit = true
+                staticProxy.removeApplication (-> process.exit())
+
+            process.on('exit', exitHandler.bind(null));
+            process.on('SIGINT', exitHandler.bind(null));
+            process.on('uncaughtException', exitHandler.bind(null));
```
Comment (nono) [inline src/main.coffee]: Why the `.bind(null)`?

File: src/static-proxy.coffee [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+        git: "https://github.com/cozy-labs/cozy-browser-shell.git"
+        icon: "main_icon.png"
+        type: "static"
+        name: BENCH_SLUG
+        slug: BENCH_SLUG
+        state: "installed"
+        permissions: All: description: "Dev mode"
+    }
+
+    console.log "CREATE APPLICATION #{BENCH_SLUG} on #{remoteCozy}"
```
Comment (nono) [inline src/static-proxy.coffee]: Maybe you can use printit for logs

File: src/static-proxy.coffee [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+        err ?= new Error(body.message) if body.error
+        callback err, body
+
+ensureApplication = (callback) ->
+    options = headers: {cookie}
+    console.log "CHECK IF #{BENCH_SLUG} IS INSTALLED ON #{remoteCozy}"
+    newClient(remoteCozy, options).get '/api/applications', (err, res, apps) ->
+        exists = apps.rows.find (app) -> app.slug is BENCH_SLUG
+        if exists
+            return callback null, exists
```
Comment (nono) [inline src/static-proxy.coffee]: Why the return here?

File: src/static-proxy.coffee (hunk 3 above)
Comment (aenario) [inline src/static-proxy.coffee]: This use multi args console.log and I am a bit in a hurry. To be fixed later.

**Comment by nono** [comment]
@aenario what is the status of this PR?

**Comment by frankrousseau** [comment]
ping @aenario

**Pull request merged**